sass/coveo-styleguide/scss/controls/_checkboxes.scss

```diff
@@ -51,24 +51,21 @@ input[type='checkbox'].coveo-checkbox {
   }
 
   &:checked + button {
-    background-color: $green;
-    border-color: $green;
-
     &:before {
       position: absolute; // Position it in the center of the box
       top: 1px;
-      left: 4px;
+      left: 5px;
 
       box-sizing: content-box;
       display: block;
       width: 4px; // Make it a small rectangle so the border will create an L-shape
-      height: 8px;
+      height: 9px;
 
       content: '';
 
-      border: solid $pure-white; // Add a white border on the bottom and left, creating that 'L'
+      border: solid $coveo-blue; // Add a white border on the bottom and left, creating that 'L'
```

Comment:
Is the comment needed or could it be removed? If needed, it should say `Add a blue border ...` 😄

Reply:
Yeah haha